Doc/library/pathlib.rst

```diff
@@ -1413,6 +1336,88 @@ example because the path doesn't exist).
 
    .. versionadded:: 3.12
 
+
+Other methods
+^^^^^^^^^^^^^
+
+Some of these methods can raise an :exc:`OSError` if a system call fails (for
```

Comment:
Isn't this also true for the directory methods that you moved to a separate section?